src/test/overlay/short_read_test.cpp

```diff
@@ -189,11 +183,15 @@ class short_read_test : public beast::unit_test::suite
                 : Child(server)
                 , server_(server)
                 , test_(server_.test_)
-                , acceptor_(test_.io_service_, endpoint())
+                , acceptor_(test_.io_service_,
+                    endpoint_type(address_type::from_string("127.0.0.1"), 0))
```

Comment:
@mellery451 I'm not following this change. Can you explain why the port changed from 9000 to 0?

Reply:
Answering my own question: ```Alternatively, it can provide port 0 to bind() as its connection parameter instead. That triggers the operating system to automatically search for and return a suitable available port in the TCP/IP dynamic port number range.```

Reply:
to allow the OS to pick any available port to bind/listen. There is no reason that I'm aware of for the test to specifically be on port 9000, so as long as the client and server both know the port that is selected, they can communicate. This prevents infrequent but occasional errors where 9000 was conflicting or not yet available for use.